src/bin/vip-import-sql.js

```diff
@@ -0,0 +1,78 @@
+#!/usr/bin/env node
+
+/**
+ * @flow
+ * @format
+ */
+
+/**
+ * External dependencies
+ */
+import chalk from 'chalk';
+
+/**
+ * Internal dependencies
+ */
+import command from 'lib/cli/command';
+import { currentUserCanImportForApp, isSupportedApp } from 'lib/site-import/db-file-import';
+import { getFileMeta, uploadFile } from 'lib/client-file-uploader';
+
+/**
+ * - Include `import_in_progress` state & error out if appropriate (this likely needs to be exposed in the data graph)
```

Comment:
We'll definitely need a query for this if we want to show the progress in CLI/Dashboard